flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/JobManagerDeploymentStatus.java

```diff
@@ -41,14 +38,15 @@
     /** JobManager deployment not found, probably not started or killed by user. */
     MISSING;
 
-    public UpdateControl<FlinkDeployment> toUpdateControl(FlinkDeployment flinkDeployment) {
+    public UpdateControl<FlinkDeployment> toUpdateControl(
+            FlinkDeployment flinkDeployment, int reconcileDelayInSec, int portCheckIntervalInSec) {
```

Comment:
If we have a dedicated configclass for all these values, then it's enough to pass that and can be changed/extended later